src/main/java/com/cloudbees/hudson/plugins/folder/computed/DeletingChildrenException.java

```diff
@@ -0,0 +1,20 @@
+package com.cloudbees.hudson.plugins.folder.computed;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DeletingChildrenException extends RuntimeException {
+
+    private List<IOException> exceptions;
```

Comment:
would you consider to store just the message?

Reply:
I stored the exceptions just in case they could be useful in the future.

Reply:
The message wouldn't be enough to understand which part of the code triggered the exception.